### PR TITLE
Updated Dockerfile for OpenvinoEP

### PR DESCRIPTION
**Pull request by avidiyal**
Signed-off-by: avidiyal <akhila.vidiyala@intel.com>

**Description**: 
-Updated Docker file for Openvino EP and also Readme. 

**Motivation and Context**

 This supports building the docker image for Openvino EP which supports accelerators CPU,GPU and  VAD-R

### CODE REVIEW

**Comment by jywu-msft** [comment]
please fix the conflict. thanks.

File: dockerfiles/README.md [hunk 1]
```diff
@@ -120,3 +178,4 @@
   ```
   curl  -X POST -d "@request.json" -H "Content-Type: application/json" http://0.0.0.0:{your_local_port}/v1/models/mymodel/versions/3:predict  
   ```
+>>>>>>> master
```
Comment (jywu-msft) [inline dockerfiles/README.md]: why is this line here?

File: dockerfiles/Dockerfile.openvino [hunk 2]
```diff
@@ -3,35 +3,65 @@
...
-ENV LD_LIBRARY_PATH $INTEL_CVSDK_DIR/deployment_tools/inference_engine/lib/ubuntu_16.04/intel64:$INTEL_CVSDK_DIR/deployment_tools/inference_engine/temp/omp/lib:/usr/local/openblas/lib:$LD_LIBRARY_PATH
 
-ENV PATH $INTEL_CVSDK_DIR/deployment_tools/model_optimizer:$PATH
-ENV PYTHONPATH $INTEL_CVSDK_DIR/deployment_tools/model_optimizer:$INTEL_CVSDK_DIR/tools:$PYTHONPATH
+RUN mkdir -p /opt/cmake/bin
+
+ENV PATH /opt/cmake/bin:$PATH
+ENV LANG en_US.UTF-8
+RUN wget https://github.com/Kitware/CMake/releases/download/v3.13.2/cmake-3.13.2-Linux-x86_64.tar.gz && \
+    tar -xf cmake-3.13.2-Linux-x86_64.tar.gz --strip 1 -C /opt/cmake
```
Comment (manashgoswami) [inline dockerfiles/Dockerfile.openvino]: remove cmake..tar.gz after this step.

File: dockerfiles/README.md [hunk 3]
```diff
@@ -83,24 +83,79 @@
...
+     ```
+       # Will be available with next release
+     ```
+
+2. Run the docker image
+    ```
+    docker run -it --device /dev/dri:/dev/dri onnxruntime-gpu:latest
+    ```
+
+## VAD-R Accelerator Version
```
Comment (manashgoswami) [inline dockerfiles/README.md]: missing steps for building MYRIAD_FP16

File: dockerfiles/README.md [hunk 4]
```diff
@@ -83,24 +83,79 @@
...
-  ```
-  # If you have a Linux machine, preface this command with "sudo"
-
-  docker run -it onnxruntime-openvino
-  ```
+1. Build the onnxruntime image for all the accelerators supported as below 
+
+   Retrieve your docker image in one of the following ways.
+
+    -  For building the docker image, download OpenVINO online installer version 2018 R5.0.1 from [here](https://software.intel.com/en-us/openvino-toolkit/choose-download) and copy the openvino tar file in the same directory and build the image. The online installer size is only 16MB and the components needed for the accelerators are mentioned in the dockerfile.
```
Comment (jywu-msft) [inline dockerfiles/README.md]: I went to that link and couldn't download 2018 R5.0.1  , they only seem to have 2019 R1 available for download
Is there another direct link to R5? otherwise, this will be a problem.

Comment (jywu-msft) [reply dockerfiles/README.md]: I found https://download.01.org/openvinotoolkit/2018_R5/packages/ 
Is it officially supported?

Comment (jywu-msft) [reply dockerfiles/README.md]: the file I downloaded from there was l_openvino_toolkit_ie_p_2018.5.445.tgz
i don't think it works with the Dockerfile.openvino since it untars to a directory called inference_engine_vpu_arm and doesn't have some required files.

Comment (avidiyal) [reply dockerfiles/README.md]: Attaching the screenshot for downloading the R5 version 
image

File: dockerfiles/README.md (hunk 3 above)
Comment (avidiyal) [reply dockerfiles/README.md]: Will update it by today

File: dockerfiles/README.md (hunk 4 above)
Comment (jywu-msft) [reply dockerfiles/README.md]: R5 is not currently an option. I just signed up this morning. See screenshot. 

Screenshot from 2019-07-11 12-43-33

File: dockerfiles/README.md [hunk 5]
```diff
@@ -83,24 +83,79 @@
...
+## VAD-R Accelerator Version 
+
+1. Retrieve your docker image in one of the following ways. 
+   - Build the docker image from the DockerFile in this repository.
+     ``` 
+      docker build -t onnxruntime-vadr --build-arg DEVICE=VAD-R_FP16 --network host . 
+     ```
+   - Pull the official image from DockerHub.
+     ```
+      # Will be available with ONNX Runtime 0.2.0
```
Comment (jywu-msft) [inline dockerfiles/README.md]: ONNX Runtime 0.2.0 seems to be a typo? (current version is 0.4.0)

File: dockerfiles/README.md (hunk 4 above)
Comment (avidiyal) [reply dockerfiles/README.md]: We are trying to resolve this issue. Will email as soon as it is fixed.

File: dockerfiles/README.md (hunk 3 above)
Comment (avidiyal) [reply dockerfiles/README.md]: I have updated these and pushed the latest change

**Comment by jywu-msft** [comment]
branch is out of date

**Comment by avidiyal** [comment]
Updated to the latest repo

File: dockerfiles/README.md (hunk 4 above)
Comment (jywu-msft) [reply dockerfiles/README.md]: any updates on this issue?
